.vib/joomla/goss/joomla.yaml

```
@@ -0,0 +1,48 @@
+command:
+  check-app-version:
+    exec: grep {{ .Env.APP_VERSION }} /opt/bitnami/joomla/administrator/manifests/files/joomla.xml
+    exit-status: 0
+  # Placeholders in SQL installation files should have been substituted
+  sql-placeholders:
+    exec: grep "#__" /opt/bitnami/joomla/installation/sql/mysql/*.sql
+    exit-status: 1
```

This is a bit of a nitpick, but I think we should try to have our commands return an `exit-status=0` when possible. In this case, we could maybe something like:

```suggestion
    exec: if [ $(grep "#__" /opt/bitnami/joomla/installation/sql/mysql/*.sql | wc -l) -gt 0 ]; then exit 1; fi
    exit-status: 0
```

The trade-off is increasing the logic complexity, so I don't have any strong thoughts on this. Another option a bit cleaner could be:

```suggestion
    exec: grep "#__" /opt/bitnami/joomla/installation/sql/mysql/*.sql | wc -l
    exit-status: 0
    stdout:
      - /^0/
```